openmc/mesh.py

```diff
@@ -799,6 +799,99 @@ def write_data_to_vtk(self, filename, datasets, volume_normalization=True):
             volume_normalization=volume_normalization
         )
 
+    def plot_tally_values_slice(
+        self,
+        dataset: np.ndarray,
+        basis: str = 'xy',
+        axes: 'matplotlib.axes.Axes' = None,
+        colorbar_label: str = None
+    ) -> 'matplotlib.axes.Axes':
+        """Maps the dataset values to the mesh and exports an image.
+
+            dataset : np.ndarray
+                A 2-D array of values to plot, this is rotated to suit the
+                MatPlotLib imshow function.
+            basis : {'xy', 'xz', 'yz'}
+                The basis directions for the plot
+            axes : matplotlib.Axes
+                Axes to draw to
+            colorbar_label : str
+                If set a color bar with the specified label will be added to the plot
+
+        Returns
+        -------
+        matplotlib.image.AxesImage
+            Resulting image
+        """
+
+        import matplotlib.pyplot as plt
+        from matplotlib.colors import LogNorm
+
+        if basis == "xy":
+            oriented_data = np.rot90(dataset, 1)
```

Comment:
should these numpy rotation operations go into the tally class as they are data processing?

Reply:
or perhaps this is specific orientation for matplotlib imshow usage

Reply:
Yeah exactly. Because this is assuming use of `imshow` it makes sense to have the rotation done here I'd say.